lib/lf/uscalfe/mesh_function_binary.h

```diff
@@ -247,6 +247,103 @@ struct OperatorSubtraction {
   }
 };
 
+struct OperatorMultiplication {
+  // Multiplication of scalar types:
```

Comment:
Sometimes it would be desirable to have a more detailed inline documentation. Students may browse this code in orer to learn advanced C++ programming and in this case  they should get some guidance.

Reply:
Okay I will add some more documentation on these structs.